functions/Get-DbaSsisExecutionHistory.ps1

```diff
@@ -49,12 +49,12 @@ function Get-DbaSsisExecutionHistory {
 
             Shows what would happen if the command were executed and would return the SQL statement that would be executed per instance.
     #>
-    [CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess = $true)]
```

Comment:
parametersets are only required when you have conflicting params, like..

"If you run Copy-DbaDatabase, you can run EITHER backuprestore OR detachattach"  or "database is ONLY mandatory when..."

SupportsShoudlProcess is for whatif and is only used when there's modification to settings/data/etc.